lib/Echidna/Transaction.hs

```diff
@@ -1,9 +1,7 @@
-{-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TemplateHaskell #-}
 {-# LANGUAGE TupleSections #-}
 {-# LANGUAGE ViewPatterns #-}
 
```

Comment:
Perhaps this module should be called Echidna.Transaction.Gen ?

Reply:
we can move it to gen in another PR i think. i want to mostly keep any internal modules in-place until we start finalizing the refactor

Reply:
it would probably just be `Echidna.Gen` rather than `Echidna.Transaction.Gen` but yeah